### PR TITLE
EOS-16027: Unboxing Restart all services once CSM and SSPL config's are updated (commit set)

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Anjali Somwanshi anjali.somwanshi@seagate.com

### CODE REVIEW

**Review by SomwanshiAnjali — APPROVED** [review]

**Review by nitin-seagate — COMMENTED** [review]
Looks good

File: cli/src/factory_ops/unboxing/init [hunk 1]
```diff
@@ -303,7 +303,7 @@ sleep 2
 echo "Bringing HCTL cluster out of maintenance mode" 2>&1 | tee -a ${LOG_FILE}
 hctl node unmaintenance --all 2>&1 | tee -a ${LOG_FILE}
 
-echo "Ensure cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
+echo "Ensuring the cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
```
Comment (nitin-seagate) [inline cli/src/factory_ops/unboxing/init:306]: Did we not changed this message - "Restarting the cluster resources through switching maintenance mode"

**Review by nitin-seagate — APPROVED** [review]

File: cli/src/factory_ops/unboxing/init [hunk 2]
```diff
@@ -329,41 +329,19 @@ provisioner pillar_set commons/repo/lustre \"\" --logfile --logfile-filename ${L
...
-echo -n "UDS...................................." | tee -a ${LOG_FILE}
-pcs resource refresh uds >> ${LOG_FILE}
-sleep 5
-echo "Ok." | tee -a ${LOG_FILE}
-
-echo -n "HAProxy................................" | tee -a ${LOG_FILE}
-pcs resource refresh haproxy-c1 >> ${LOG_FILE}
-pcs resource refresh haproxy-c2 >> ${LOG_FILE}
-echo "Ok." | tee -a ${LOG_FILE}
+echo "Restarting the Cortx Services" | tee -a ${LOG_FILE}
```
Comment (nitin-seagate) [inline cli/src/factory_ops/unboxing/init:332]: Did we not changed this message - "Restarting the cluster resources through switching maintenance mode"

**Review by nitin-seagate — APPROVED** [review]

**Review by 83bhp — APPROVED** [review]

File: cli/src/factory_ops/unboxing/init (hunk 2 above)
Comment (nitin-seagate) [reply cli/src/factory_ops/unboxing/init:332]: Yes, I see the change not pushed to pre-cortx-1.0 as well

**Review by ujjwalpl — APPROVED** [review]

**Pull request merged**